.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -200,16 +200,20 @@ data class CIBuildModel(
 
     init {
         val subprojectMap = subProjects.map { it.name to it }.toMap()
-        val buckets = listOf(
-            listOf("resources", "resourcesGcs", "resourcesHttp", "resourcesS3", "resourcesSftp"),
-            listOf("platformBase", "platformJvm", "platformNative")
+        val buckets = mapOf(
+            "resources" to listOf("resources", "resourcesGcs", "resourcesHttp", "resourcesS3", "resourcesSftp"),
+            "platformBase" to listOf("platformBase", "platformJvm", "platformNative"),
+            "bucket1" to listOf("kotlinDslProviderPlugins", "buildCachePackaging", "native", "snapshots", "internalPerformanceTesting", "internalIntegTesting", "execution", "publish", "ear", "languageJvm"),
```

Comment:
Let's separate the ID from the name and use all the subprojects as name. For the id, `bucket1` is fine I suppose. The name should contain the list of subprojects.

Reply:
PTAL!